### PR TITLE
Fix url to apidocs

### PR DESCRIPTION
**Pull request by erikjanwestendorp**
Fix url to apidocs in the **[Embedded Media Provider](https://our.umbraco.com/documentation/Extending/Embedded-Media-Provider/)**  article.

### CODE REVIEW

File: Extending/Embedded-Media-Provider/index.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ The list of available default Embed Providers in an Umbraco install are as follo
 * Hulu
 * Giphy
 
-You can see the details of these, and any recent editions in the C# developer reference for [Umbraco.Web.Media.EmbedProviders](https://our.umbraco.com/apidocs/v9/csharp/api/Umbraco.Cms.Core.Media.EmbedProviders.html).
+You can see the details of these, and any recent editions in the C# developer reference for [Umbraco.Web.Media.EmbedProviders](https://apidocs.umbraco.com/v9/csharp/api/Umbraco.Cms.Core.Media.EmbedProviders.html).
```
Comment (eshanrnh) [inline Extending/Embedded-Media-Provider/index.md:41]:
```suggestion
You can see the details of these, and any recent editions in the C# developer reference for [Umbraco.Web.Media.EmbedProviders](https://apidocs.umbraco.com/v10/csharp/api/Umbraco.Cms.Core.Media.EmbedProviders.html).
```

**Review by eshanrnh — APPROVED** [review]
Looks great, @erikjanwestendorp 🙌 Sharp eyes for detail.

I just added a small suggestion to point to v10 API doc as the latest version is version 10 🙂

**Comment by erikjanwestendorp** [comment]
@eshanrnh Thanks for your review 😄. I was already wondering if I should point to v9 or 10 because the article it self targets both. Just updated the PR with your suggestion.

**Comment by eshanrnh** [comment]
That's great 💪 I'll merge it in 😃

**Pull request merged**